src/couch_epi_data.erl

```diff
@@ -88,7 +88,8 @@ handle_call(stop, _From, State) ->
 handle_call(_Request, _From, State) ->
     {reply, ok, State}.
 
-handle_cast(init, #state{pending = Pending} = State) ->
+handle_cast(init, #state{pending = Pending, handle = Handle} = State) ->
+    couch_epi_module_keeper:maybe_start_keeper(couch_epi_data_gen, Handle),
```

Comment:
a cast that starts a supervised child? Is it not possible to get the ordering right during application start?